src/collection-browser.ts

```diff
@@ -2562,11 +2564,9 @@ export class CollectionBrowser
           display: flex;
           justify-content: space-between;
           align-items: flex-start;
-          margin: 10px 0;
         }
 
         .desktop #facets-header-container {
-          padding-top: 2rem;
```

Comment:
Currently this is the head element on facets area. so instead of apply on internal element, `.column` class will have padding-top: 2rem for facets and collection-browser sections.